Add New Product Functionality

apoorvdtu opened the pull request.

apoorvdtu committed:
basic setup

apoorvdtu committed:
react setups

apoorvdtu committed:
Added AddProduct Page

apoorvdtu committed:
Delete .parcel-cache directory

apoorvdtu committed:
Delete dist directory

apoorvdtu committed:
Replaced fetchLocalStorage function to hooks

apoorvdtu committed:
Delete node_modules directory

bansalshruti69 commented on index.jsx:
```diff
@@ -0,0 +1,6 @@
+import {createRoot} from 'react-dom/client'
+import App from './App.jsx'
+import 'bootstrap/dist/css/bootstrap.min.css';
```
why are we importing css here?

apoorvdtu committed:
Refactored Code & added product Reducer

14Feb:added productReducer and  addProductFormValidation
14Feb:added productReducer
14Feb:added productReducer  Please enter the commit message for your changes. Lines starting
14Feb:added productReducer  with '#' will be ignored, and an empty message aborts the commit.
14Feb:added productReducer
14Feb:added productReducer  Author:    Apoorv Ambavat <77097580+apoorvdtu@users.noreply.github.com>
14Feb:added productReducer  Date:      Tue Feb 13 22:13:43 2024 +0530
14Feb:added productReducer
14Feb:added productReducer  interactive rebase in progress; onto c0b7038
14Feb:added productReducer  Last commands done (2 commands done):
14Feb:added productReducer     pick 45e4b4b Refactored Code & added product Reducer
14Feb:added productReducer     squash 3d5d41e Added addProductFormValidation
14Feb:added productReducer  No commands remaining.
14Feb:added productReducer  You are currently rebasing branch 'ecom' on 'c0b7038'.
14Feb:added productReducer
14Feb:added productReducer  Changes to be committed:
14Feb:added productReducer 	new file:   .eslintignore
14Feb:added productReducer 	new file:   .eslintrc.js
14Feb:added productReducer 	new file:   .prettierignore
14Feb:added productReducer 	new file:   .prettierrc
14Feb:added productReducer 	new file:   .vscode/settings.json
14Feb:added productReducer 	modified:   App.jsx
14Feb:added productReducer 	modified:   components/addproduct/AddProduct.jsx
14Feb:added productReducer 	modified:   components/addproduct/AddProductForm.jsx
14Feb:added productReducer 	deleted:    components/addproduct/AddProductHeader.jsx
14Feb:added productReducer 	deleted:    components/cart/Cart.jsx
14Feb:added productReducer 	deleted:    components/cart/Cart.module.css
14Feb:added productReducer 	deleted:    components/cart/CartHeader.jsx
14Feb:added productReducer 	deleted:    components/cart/CartItem.jsx
14Feb:added productReducer 	deleted:    components/cart/CartPrice.jsx
14Feb:added productReducer 	modified:   components/utilities/Product.jsx
14Feb:added productReducer 	modified:   components/utilities/constant.js
14Feb:added productReducer 	deleted:    icons/Appliances.png
14Feb:added productReducer 	deleted:    icons/Beauty.png
14Feb:added productReducer 	deleted:    icons/Fashion.png
14Feb:added productReducer 	deleted:    icons/Flights.png
14Feb:added productReducer 	deleted:    icons/FlipkartNavbarLogo.png
14Feb:added productReducer 	deleted:    icons/Furniture.png
14Feb:added productReducer 	deleted:    icons/Grocery.png
14Feb:added productReducer 	deleted:    icons/HomeKitchen.png
14Feb:added productReducer 	deleted:    icons/MobileCategoryIcon.png
14Feb:added productReducer 	deleted:    icons/TVElectronics.png
14Feb:added productReducer 	deleted:    icons/becomeseller.png
14Feb:added productReducer 	deleted:    icons/login.png
14Feb:added productReducer 	deleted:    icons/phoneicon.png
14Feb:added productReducer 	deleted:    icons/searchicon.svg
14Feb:added productReducer 	deleted:    icons/shoppingcart.png
14Feb:added productReducer 	modified:   index.jsx
14Feb:added productReducer 	modified:   package-lock.json
14Feb:added productReducer 	modified:   package.json
14Feb:added productReducer
